### PR TITLE
Rewrite of getExtension

### PR DESCRIPTION
**Pull request by qligier**
It uses now parse_url and pathinfo.

### CODE REVIEW

**Comment by electerious** [comment]
Perfect! :)

File: php/Modules/Photo.php [hunk 1]
```diff
@@ -844,8 +844,8 @@ public function getArchive() {
 		}
 
 		// Get extension
-		$extension = getExtension($photo->url);
-		if ($extension===false) {
+		$extension = getExtension($photo->url, true);
```
Comment (electerious) [inline php/Modules/Photo.php]: The field is called URL, but it just contains the filename (e.g. 4a3a4b31c88b420c29a007a8db6dd42f.gif). Should be changed to `getExtension($photo->url, false)`, right?

File: php/Modules/Album.php [hunk 2]
```diff
@@ -278,7 +278,7 @@ public function getArchive() {
 			if (!@is_readable($photo->url)) continue;
 
 			// Get extension of image
-			$extension = getExtension($photo->url);
+			$extension = getExtension($photo->url, true);
```
Comment (electerious) [inline php/Modules/Album.php]: Same as https://github.com/electerious/Lychee/pull/488/files#r56469924

**Comment by qligier** [comment]
Yes, it works anyway but the second parameter should be false in both cases, sorry for that.

**Pull request merged**